packages/common/vite.config.ts

```diff
@@ -9,7 +9,13 @@ export default defineConfig({
       formats: ['es'],
     },
   },
-  plugins: [dts({ rollupTypes: true, tsconfigPath: 'tsconfig.build.json' })],
```

Comment:
Had to remove rollupTypes due to:

<img width="907" alt="Skärmavbild 2024-03-21 kl  14 24 48" src="https://github.com/tidal-music/tidal-sdk-web/assets/877455/af5665a0-f0e5-4536-8f31-55d62375784a">

Reply:
https://github.com/microsoft/rushstack/issues/3614